src/serving/runners.py

```diff
@@ -0,0 +1,38 @@
+import bentoml
+
+class ModelRunner:
+    """
+    Runner class for model serving
+    """
+    
+    def __init__(self, model_name, model_version = "latest"):
```

Comment:
argument에도 타입형힌트 추가 부탁드려요~

```suggestion
    def __init__(self, model_name: str, model_version: str = "latest"):
```

Reply:
모든 클래스, 함수에 대해 타입형 힌트 추가했습니다!